src/Core/Checkout/Order/Aggregate/OrderDelivery/OrderDeliveryStates.php

```diff
@@ -11,7 +11,9 @@ final class OrderDeliveryStates
     public const STATE_OPEN = 'open';
     public const STATE_PARTIALLY_SHIPPED = 'shipped_partially';
     public const STATE_SHIPPED = 'shipped';
-    public const STATE_RETURNED = 'returned';
+    public const STATE_PARTIALLY_DELIVERED = 'delivered_partially';
```

Comment:
Hi @lxregistry, we already have state  STATE_PARTIALLY_SHIPPED and STATE_SHIPPED for delivery status. What is the reason behind you want to add 2 other states?

Reply:
Please add delivered status, like on ebay, 
shipped date, delivered date, an order could be shipped by not delivered, lost package